cmd/tink-controller/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine:3.15
+ENTRYPOINT ["/usr/bin/tink-controller"]
+EXPOSE 42113
+EXPOSE 42114
+
+ARG TARGETARCH
+ARG TARGETVARIANT
+
+RUN apk add --update ca-certificates && \
+    apk add --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing cfssl
```

Comment:
do we need cfssl in `tink-controller`? I'm not seeing where its used in this PR.

Reply:
Good call, probably not.  Just copy/pasted from elsewhere